### ISSUE TITLE
need a research paper assistant

### ISSUE THREAD

**Issue by jmanhype** [yacineMTB/talk#51]
Would love to have this be able to talk to me about a research paper I load into a vector database or similar functionality. I have some custom made txt based research assistant but a voice one would be extremely useful

**Comment by jmanhype** [comment]
@cto-junior  @choombaa  @cpuchner  @yacineMTB

**Comment by choombaa** [comment]
Good idea, I would suggest loading it in as either a persona or a hardcoded prompt!

**Comment by jmanhype** [comment]
So the research assistant part is easy but im talking give the persona access to postgres vector db and do some functions tobmakebsure it truncates the text so it doesn't go over the window this would make it very powerful

**Comment by jmanhype** [comment]
Use this to derive functionality from. Like hook this up to talk please https://github.com/daveshap/Quickly_Extract_Science_Papers

**Comment by jmanhype** [comment]
Lmk
